tutorials/measurement_chain.ipynb

```diff
@@ -0,0 +1,553 @@
+{
```

Comment:
<p>lacking a comma: <span class="markdownDiffSpan" style="background: rgb(209, 255, 221) none repeat scroll 0% 0%; display: inline-block; --darkreader-inline-bgcolor:#114627; --darkreader-inline-bgimage:none;" data-darkreader-inline-bgcolor="" data-darkreader-inline-bgimage="">we demonstrated, required </span></p><p><span class="markdownDiffSpan" style="background: rgb(209, 255, 221) none repeat scroll 0% 0%; display: inline-block; --darkreader-inline-bgcolor:#114627; --darkreader-inline-bgimage:none;" data-darkreader-inline-bgcolor="" data-darkreader-inline-bgimage="">While this is method is -&gt; While this method is</span></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Sure about the comma? I am not and grammrly doesn't complain either.</p>

Reply:
<p>I guess you're right :)</p>